### PR TITLE
MG-2457 - Update groups tests

### PR DESCRIPTION
**Pull request by felixgateru**
# What type of PR is this?
This is a feature as it adds tests to the groups service.

## What does this do?
This pr adds tests for the transport, service and repo layer for the groups service.

## Which issue(s) does this PR fix/relate to?

- Related Issue #2457 

## Have you included tests for your changes?
Yes, the changes are to the test files.

## Did you document any new/modified feature?
No,

### Notes
To be merged after #2506

### CODE REVIEW

File: groups/status.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ const (
 	EnabledStatus Status = iota
 	// DisabledStatus represents disabled Group.
 	DisabledStatus
-	// DeletedStatus
+	// DeletedStatus.
```
Comment (dborovcanin) [inline groups/status.go]: Either fix the linter or the comment.

File: groups/roleoperations.go [hunk 2]
```diff
@@ -140,7 +141,7 @@ func NewRolesOperationPermissionMap() map[svcutil.Operation]svcutil.Permission {
 
 const (
 	// External Permission
-	// Domains
+	// Domains.
```
Comment (dborovcanin) [inline groups/roleoperations.go]: The same, check on the linter or make a full sentence comment.

File: groups/postgres/groups.go [hunk 3]
```diff
@@ -994,7 +997,7 @@ func toDBGroupPageMeta(pm mggroups.PageMeta) (dbGroupPageMeta, error) {
 	}, nil
 }
 
-// ToDo: check and remove field "Level" after new auth stabilize
+// ToDo: check and remove field "Level" after new auth stabilize.
```
Comment (dborovcanin) [inline groups/postgres/groups.go]: Remove TODO comments.

File: groups/postgres/groups.go [hunk 4]
```diff
@@ -626,6 +628,7 @@ func (repo groupRepository) RetrieveChildrenGroups(ctx context.Context, domainID
 		query = query + " AND " + levelCondition
 	}
 
+	fmt.Println(startLevel, endLevel)
```
Comment (dborovcanin) [inline groups/postgres/groups.go]: Remove.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: groups/service.go [hunk 5]
```diff
@@ -109,6 +108,7 @@ func (svc service) ViewGroup(ctx context.Context, session mgauthn.Session, id st
 	if err != nil {
 		return Group{}, errors.Wrap(svcerr.ErrViewEntity, err)
 	}
+	group.Path = ""
```
Comment (arvindh123) [inline groups/service.go]: Why path is not send back ?

**Review by arvindh123 — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**